src/CTA.WebForms2Blazor/Helpers/ControlHelpers/EmbeddedCodeReplacers.cs

```diff
@@ -5,30 +5,40 @@ namespace CTA.WebForms2Blazor.Helpers.ControlHelpers
 {
     public static class EmbeddedCodeReplacers
     {
+        public const string EmbeddedExpressionRegexGroupName = "expr";
+        public const string DirectiveNameRegexGroupName = "DirectiveName";
+
         //DataBind regex might have issue of taking front parentheses
```

Comment:
Same thing with these a comment above and a sample in the comment would be helpful.